api/src/migrations/1575658419248-fix-case-insensitive-collation.ts

```
@@ -14,22 +14,39 @@ export class fixCaseInsensitiveCollation1575658419248 implements MigrationInterf
         break;
+      case DbType.POSTGRES:
+        console.log('PostgreSQL uses case-sensitive collation by default, no changes needed');
+        // PostgreSQL is case-sensitive by default, no action needed
+        break;
+      case DbType.BETTER_SQLITE3:
+        console.log('SQLite uses case-sensitive collation by default, no changes needed');
+        // SQLite is case-sensitive by default, no action needed
+        break;
       default:
-        console.log('Unknown DB type');
+        throw new Error('Unknown DB type: ' + config.db.default.type);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)